sapi/fuzzer/fuzzer-mbstring.c

```diff
@@ -51,7 +51,8 @@ int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
 		return 0;
 	}
 
-	char *Result = php_mb_convert_encoding_ex((char *) Data, Size, ToEncoding, FromEncoding, NULL);
+	size_t output_len;
+	char *Result = php_mb_convert_encoding_ex((char *) Data, Size, ToEncoding, FromEncoding, &output_len);
```

Comment:
Should be adjusted for zend_string return?

Reply:
Uhhh... yeah, it should.